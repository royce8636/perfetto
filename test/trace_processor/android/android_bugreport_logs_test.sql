select
  (select count(*) from android_logs) as cnt,
  ts,
  prio,
  tag,
  msg
from android_logs
order by ts desc
limit 10;
