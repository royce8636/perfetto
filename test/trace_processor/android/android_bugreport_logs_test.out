"cnt","ts","prio","tag","msg"
33073,1629848143333617000,4,"crash_dump64","performing dump of process 733 (target tid = 733)"
33073,1629848143333382000,4,"tombstoned","found intercept fd 512 for pid 733 and type kDebuggerdNativeBacktrace"
33073,1629848143333341000,4,"tombstoned","received crash request for pid 733"
33073,1629848143332905000,4,"crash_dump64","obtaining output fd from tombstoned, type: kDebuggerdNativeBacktrace"
33073,1629848143317213000,4,"","c1      1 init: Untracked pid 8530 exited with status 0"
33073,1629848143316238000,4,"","c1      1 init: Untracked pid 8528 exited with status 0"
33073,1629848143314679000,4,"libc","Requested dump for tid 733 (Binder:733_2)"
33073,1629848143314401000,4,"tombstoned","registered intercept for pid 733 and type kDebuggerdNativeBacktrace"
33073,1629848143313939000,4,"dumpstate","libdebuggerd_client: started dumping process 733"
33073,1629848143312890000,5,"tombstoned","missing output fd"
